jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/Dispatcher.java

```diff
@@ -645,26 +655,13 @@ public String getQueryString()
         @Override
         public String getRequestURI()
         {
-            return _uri == null ? null : _uri.getPath();
-        }
-
-        @Override
-        public Object getAttribute(String name)
```

Comment:
All of the `RequestDispatcher.ERROR*` attributes already exist at this point in time, there's no point doing it again at this level.

Also, many of the test cases in ErrorPageTest use those same attributes for assertions, proving that they are still present.

Reply:
The `RequestDispatcher.ERROR_*` attributes are being set from 2 places before this `Dispatcher$ErrorRequest` is created.

* `org.eclipse.jetty.ee10.servlet.ServletRequestState.sendError(int code, String message)`
* `org.eclipse.jetty.ee10.servlet.ServletRequestState.sendError(Throwable th)`

If we want to mimic how `Dispatcher$ForwardRequest` attributes are handled in a future PR sure, but that's a bigger change that is out of scope for this PR.

Reply:
It is interesting that the attributes for forwarding and including are "set" by their request wrappers, but for error handling we still set the attributes on the actual request rather than just adding them via a wrapper.
That might be in part because we want to log some of them?  But it is probably mostly that we didn't get around to converting to the wrapping style.

I'm not sure we need to change this... just something to be aware of.... maybe put a comment in saying how/why it is different.

Reply:
I opened an issue on this at #10129 so we can review this behavior after 12.0.0 release.